bin/compile

```diff
@@ -109,12 +107,11 @@ export BLACKFIRE_ENDPOINT=\${BLACKFIRE_ENDPOINT:-"https://blackfire.io"}
 export BLACKFIRE_AGENT_SOCKET="${BLACKFIRE_AGENT_SOCKET}"
 
 if [ ! -S ${BF_AGENT_SOCKET_FILE} ] ; then
-  blackfire-agent \
+  blackfire agent:start \
       --config=/app/${BF_AGENT_CONFIG_FILE} \
       --log-file="/app/${BF_AGENT_LOG_DIR}/agent.log" &
 fi
 EOF
 
-topic "Blackfire agent and CLI tool are now installed."
-${BF_BUILD_BIN_DIR}/blackfire-agent -v | indent
+topic "Blackfire CLI command is now installed."
```

Comment:
just `Blackfire is now installed.` or `Blackfire command is now installed.` instead?

Reply:
It was just a suggestion, I had approved the PR ;-)